hr_payslip_line_manually/models/hr_payslip.py

```diff
@@ -0,0 +1,62 @@
+from odoo import fields, models
+
+
+class HrPayslip(models.Model):
+    _inherit = "hr.payslip"
+
+    line_manually_ids = fields.One2many(
+        "hr.payslip.line.manually",
+        "res_id",
+        string="Manual Payslip Lines",
+        domain=[("model", "=", "hr.payslip")],
+        context={"default_model": "hr.payslip"},
+        copy=True,
+    )
+
+    def _compute_payslip_line(self, rule, localdict, lines_dict):
```

Comment:
I think you can just do this here: 

`    def _compute_payslip_line(self, rule, localdict, lines_dict):
        localdict, lines_dict = super()._compute_payslip_line(rule, localdict, lines_dict)
        values = localdict["result_rules"].dict.get[rule.code]
        current_key = (rule.code+ "-"+ str(localdict["contract"].id))
        new_key = (rule.code + "-" + str(localdict["contract"].id) + "-" + str(values.get("analytic_account_id")))
        lines_dict[new_key] = lines_dict.pop(current_key)
        lines_dict[new_key].dict["analytic_account_id"] = values.get("analytic_account_id")
        return localdict, lines_dict`

I didn't test it but surely you can do this without copying the whole method, just edit the returning values and the values you can get from "result_rules" you don't need to calculate them again.

Reply:
Sorry gitgub broken my indentation. Please copy the code to python file to read it easily.